Cesium3DTilesSelection/src/Tileset.cpp

```diff
@@ -367,7 +367,7 @@ void Tileset::notifyTileUnloading(Tile* pTile) noexcept {
   }
 }
 
-uint32_t Tileset::computeLoadProgress() noexcept {
+float_t Tileset::computeLoadProgress() noexcept {
```

Comment:
```suggestion
float Tileset::computeLoadProgress() noexcept {
```

Not sure I'm completely familiar with the difference between `float` and `float_t`, but I think it's more consistent with the rest of the codebase to stick with float. From my quick googling of the difference just now, it appears platforms are allowed to implement `float_t` as double-precision, which forces the compiler to have to treat all intermediary results involving `float_t` as doubles, just in case. I think `float` on the other hand is guaranteed to have a size of 4 bytes. Doesn't particularly matter in this case, but it's good to be consistent regardless.

This is what I was reading on the difference between the two: https://stackoverflow.com/questions/5390011/whats-the-point-of-float-t-and-when-should-it-be-used